### ISSUE TITLE
Lighthouse Report - Colour Contrast

### ISSUE THREAD

**Issue by BrentM2005** [iraana/Capstone-Project#87]
Colour contrast is also a big part of accessibility.  Here are the elements that need to be changed:

- Selected menu day color contrast (Menu.tsx)
- Pickup Location text contrast (AboutPage.tsx)
- Submit review button needs a better colour contrast (Reviews.tsx)
- Make button subheadings black (AdminHome.tsx)
- Numerous elements don't contrast well enough (Analytics.tsx)
- Numerous elements don't contrast well enough (PendingOrders.tsx / ArchivedOrders.tsx / CancelledOrders.tsx) (Light and Dark modes)
- Edit Item doesn't contrast well (AddMenu.tsx)
- Edit item doesn't contrast well (ListDishes.tsx)
- In date range "to" should be black instead of gray (AdminReviews.tsx)
- Export to CSV doesn't contrast well enough (AdminReviews.tsx)

**Comment by Sujit-sault** [comment]
All color contrast issues have been fixed. Changes pushed to branch `87-lighthouse-report---colour-contrast` and ready for review.
